### PR TITLE
Add error message to GridIn::ExcNoTriangulationSelected

### PR DESCRIPTION
**Pull request by peterrum**

### CODE REVIEW

File: include/deal.II/grid/grid_in.h [hunk 1]
```diff
@@ -754,7 +754,12 @@ class GridIn
   /**
    * Exception
    */
-  DeclException0(ExcNoTriangulationSelected);
+  DeclExceptionMsg(ExcNoTriangulationSelected,
+                   "No Triangulation has been attached to this GridIn object "
+                   "so that it cannot filled during any read function calls. "
```
Comment (bangerth) [inline include/deal.II/grid/grid_in.h:759]:
```suggestion
                   "so that nothing can be filled during any read function calls. "
```

**Review by bangerth — COMMENTED** [review]
Related to my old hobby horse #611.

File: include/deal.II/grid/grid_in.h (hunk 1 above)
Comment (peterrum) [reply include/deal.II/grid/grid_in.h]: Done :+1:

**Review by bangerth — APPROVED** [review]

**Pull request merged**